### DISCUSSION TITLE
Connected Components

### DISCUSSION THREAD

**Discussion by dma-neves** [gunrock/gunrock#1063] [Q&A] [answered]
Hi, I saw in the paper _Gunrock: A High-Performance Graph Processing Library on the GPU_ that one of the benchmarked algorithms was Connected Components, however I can't seem to find the algorithm in the source code and saw that it is no longer supported in "essentials". Is there still a way to run this algorithm? Or only by checking-out to older commits maybe?

**Comment by neoblizz** [comment, marked as answer]
Hi, it was not ported to the latest version, you can find it in the older versions only.
